### ISSUE TITLE
Web component: Tabs extended - with media - selected attribute is announced for all the tabs

### ISSUE THREAD

**Issue by Praveen-sr** [carbon-design-system/carbon-for-ibm-dotcom#5814]
## Detailed description

On JAWS FF
Issue:
Selected attribute is announced for all the tabs. 
Note: Same issue is present in "tabs extended as well" 

Expected:
Screen reader should announce the selected attribute only for the tab which is currently selected. Use aria-selected ="true" only for the currently selected tab.
